api_scripts/python/robust.py

```diff
@@ -38,18 +38,26 @@
 # default values is `100`
 @click.option('--per_page', help='How many courses to fetch per page. [default: 100]',
     default=100, type=int, envvar='CANVAS_PER_PAGE')
-def view_course_report(url, token, sort_by, per_page):
+def view_course_report(url, sort_by, per_page):
+
+    # ensure access token is available
+    TOKEN = os.environ.get('CANVAS_ACCESS_TOKEN')
+    if TOKEN == None:
+        print("No access token found. Please set `CANVAS_ACCESS_TOKEN`")
```

Comment:
I think it would be better to do 
```
TOKEN = os.environ.get('CANVAS_ACCESS_TOKEN')
while not TOKEN:
    TOKEN = click.prompt("Enter your access token", hide_input=True)
```

Reply:
Good idea. Fixed in fd29f11.